template/python3-http/Dockerfile

```diff
@@ -0,0 +1,48 @@
+FROM python:3.6-alpine
+
+ARG ADDITIONAL_PACKAGE
+# Alternatively use ADD https:// (which will not be cached by Docker builder)
+RUN apk --no-cache add curl ${ADDITIONAL_PACKAGE} \
```

Comment:
As a user, I'd like to be able to install build dependencies and have them removed after pip installs dependencies.

Is that possible?

Reply:
Some build dependencies generate .so libraries which are needed in the deployment / runtime container.